hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py

```diff
@@ -667,13 +667,20 @@ def checkEncoders(self, jname='all', option=''):
             self.rh_svc.power('all', SWITCH_OFF)
             return 0
 
+        is_result_hw = True
         print self.configurator_name, 'calib-joint ' + jname + ' ' + option
-        self.rh_svc.initializeJointAngle(jname, option)
+        is_result_hw = is_result_hw and self.rh_svc.initializeJointAngle(jname, option)
         print self.configurator_name, 'done'
-        self.rh_svc.power('all', SWITCH_OFF)
+        is_result_hw = is_result_hw and self.rh_svc.power('all', SWITCH_OFF)
         self.goActual()  # This needs to happen before turning servo on.
         time.sleep(0.1)
-        self.rh_svc.servo(jname, SWITCH_ON)
+        is_result_hw = is_result_hw and self.rh_svc.servo(jname, SWITCH_ON)
```

Comment:
I confirmed that reverting this single line resolves the issue https://github.com/tork-a/rtmros_nextage/issues/227 but I'm not sure why.
@emijah Could you take a look at this?